src/java.base/aix/classes/sun/nio/ch/Pollset.java

```diff
@@ -120,5 +120,7 @@ public static native int pollsetPoll(int pollset, long pollAddress, int numfds,
     public static native void socketpair(int[] sv) throws IOException;
     public static native void interrupt(int fd) throws IOException;
     public static native void drain1(int fd) throws IOException;
+    public static native void drain(int fd) throws IOException;
     public static native void close0(int fd);
+    public static native void configureBlocking(int fd, boolean blocking) throws IOException;
```

Comment:
I assume you don't need this, instead use IOUtil.configureBlocking(fd, blocking).

Reply:
I tried this first, but `IOUtil.configureBlocking(fd, blocking)` needs a `FileDescriptor` object instead of an `int`:
`public static native void configureBlocking(FileDescriptor fd, boolean blocking)`
And I did not find any public FileDescriptor constructor taking an `int`.
I would be happy for any hint to avoid this double implementation of configureBlocking

Reply:
> I tried this first, but `IOUtil.configureBlocking(fd, blocking)` needs a `FileDescriptor` object instead of an `int`: `public static native void configureBlocking(FileDescriptor fd, boolean blocking)` And I did not find any public FileDescriptor constructor taking an `int`. I would be happy for any hint to avoid this double implementation of configureBlocking

IOUtil.newFD is there if you need it.